xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAE.h

```diff
@@ -232,51 +232,51 @@ class CActiveAE : public IAE, public IDispResource, private CThread
   friend class CSoundPacket;
   friend class CActiveAEBufferPoolResample;
   CActiveAE();
-  virtual ~CActiveAE();
-  virtual bool  Initialize();
+  ~CActiveAE() override;
+  bool  Initialize() override;
 
 public:
-  virtual void   Shutdown();
-  virtual bool   Suspend();
-  virtual bool   Resume();
-  virtual bool   IsSuspended();
-  virtual void   OnSettingsChange(const std::string& setting);
-
-  virtual float GetVolume();
-  virtual void  SetVolume(const float volume);
-  virtual void  SetMute(const bool enabled);
-  virtual bool  IsMuted();
-  virtual void  SetSoundMode(const int mode);
+  void   Shutdown() override;
```

Comment:
please also drop the vertical alignment. I always do this when touching things like this.